src/main/java/me/zeroeightsix/kami/module/modules/movement/ElytraFlight.kt

```diff
@@ -389,21 +421,22 @@ class ElytraFlight : Module() {
         } else downSpeedControl.value.toDouble()
 
         /* Hover */
-        if (hoverTarget < 0.0 || moveUp) hoverTarget = mc.player.posY else if (moveDown) hoverTarget = mc.player.posY - calcDownSpeed
-        hoverState = (if (hoverState) mc.player.posY < hoverTarget else mc.player.posY < hoverTarget - 0.1) && altitudeHoldControl.value
+        if (!pathfind) {
+            if (hoverTarget < 0.0 || moveUp) hoverTarget = mc.player.posY else if (moveDown) hoverTarget = mc.player.posY - calcDownSpeed
+        }
+        hoverState = (if (hoverState) mc.player.posY < hoverTarget else mc.player.posY < hoverTarget - 0.1) && (altitudeHoldControl.value || pathfind)
 
         /* Set velocity */
         if (!isStandingStillH || moveUp) {
             if ((moveUp || hoverState) && (currentSpeed >= 0.8 || mc.player.motionY > 1.0)) {
-                upwardFlight(currentSpeed, getYaw())
+                upwardFlight(currentSpeed, yaw)
             } else if (!isStandingStillH || moveUp) { /* Runs when pressing wasd */
                 packetPitch = forwardPitch.value.toFloat()
                 mc.player.motionY = -fallSpeedControl.value.toDouble()
-                setSpeed(getYaw(), speedControl.value, moveUp)
+                setSpeed(yaw, speedControl.value, moveUp)
```

Comment:
Why was this change made?

Reply:
yaw is determined by the player if you are not path finding and by the algorithm when you are path finding (see 408-413)